config.yaml

```diff
@@ -19,10 +19,10 @@ options:
     type: string
     default: ""
     description: "Zuul change."
-  ppy-mirror:
```

Comment:
leave the old option as well for a while, but make the new one take precedence over the old. That way we are backwards compatible, until whomever uses this updates their configs.